FoodBookApp/UI/Views/CreateReview/CreateReview1View.swift

```diff
@@ -62,8 +66,24 @@ struct CreateReview1View: View {
                                     if (DBManager().draftExists(spot: spotId)) {
                                         DBManager().deleteDraft(spot: spotId)
                                     }
-                                    DBManager().addDraft(spotValue: spotId, cat1Value: selectedCats.indices.contains(0) ? selectedCats[0] : "", cat2Value: selectedCats.indices.contains(1) ? selectedCats[1] : "", cat3Value: selectedCats.indices.contains(2) ? selectedCats[2] : "", cleanlinessValue: cleanliness, waitTimeValue: waitingTime, foodQualityValue: foodQuality, serviceValue: service, titleValue: title, contentValue: content, uploadValue: false)
+                                    let imageName = "\(spotId).jpg"
+                                    let path = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0].appendingPathComponent(imageName)
+                                    
+                                    DBManager().addDraft(spotValue: spotId, cat1Value: selectedCats.indices.contains(0) ? selectedCats[0] : "", cat2Value: selectedCats.indices.contains(1) ? selectedCats[1] : "", cat3Value: selectedCats.indices.contains(2) ? selectedCats[2] : "", cleanlinessValue: cleanliness, waitTimeValue: waitingTime, foodQualityValue: foodQuality, serviceValue: service, imageValue: selectedImage != nil ? imageName : "", titleValue: title, contentValue: content, uploadValue: false)
+                                    
                                     isNewReviewSheetPresented.toggle()
+                                    
+                                    if selectedImage != nil {
+                                        do {
+                                            try self.selectedImage?.jpegData(compressionQuality: 1)?.write(to: path)
+                                        }
+                                        catch {
+                                            print("Error saving image: \(error.localizedDescription)")
+                                        }
+                                    }
+                                    else {
+                                        DBManager().deleteImage(spot: spotId)
```

Comment:
Remember `deleteImage()` checks if an image exists for a spot, this is not done directly in this view. This is done so if a new draft doesn't have an image it will delete the previous one (if found) from the storage